buffer/src/main/java/io/netty/buffer/PoolArena.java

```diff
@@ -190,7 +190,7 @@ private void tcacheAllocateNormal(PoolThreadCache cache, PooledByteBuf<T> buf, f
         }
     }
 
-    // Method must be called inside synchronized(this) { ... } block
+    // Method must be called inside synchronized(this) { ... } block
```

Comment:
let's put a proper `assert` with monitor check

Reply:
Lets do this as a follow up as its not related to the PR